remix/app1/webpack.browser.js

```diff
@@ -0,0 +1,207 @@
+import * as path from "node:path";
+
+import {readConfig} from "@remix-run/dev/dist/config.js";
+import {EsbuildPlugin} from "esbuild-loader";
+import {toManifest, writeManifest} from "./utils/manifest.js";
+import {default as Enhanced} from '@module-federation/enhanced'
+import DelegatesModulePlugin from '@module-federation/utilities/src/plugins/DelegateModulesPlugin.js'
+const {ModuleFederationPlugin, AsyncBoundaryPlugin} = Enhanced
+class HoistContainerReferences {
```

Comment:
-Move this back into module-federation/enhanced
-Deprecate whatever im doing in Next.js to accomplish this. 
-Rewrite it again